### ISSUE TITLE
Hyperparameter tuning of ML models of Cervical Cancer Risk Classification Notebook for 99%+ Accuracy

### ISSUE THREAD

**Issue by nash1o7** [kritikaparmar-programmer/HealthCheck#217]
Hi @kritikaparmar-programmer  @utkarsh0702 

## Proposal

I am a GSSOC 2021 Participant. After hyperparameter tuning of ML models (Logistic Regression,SVC,Random forest,Adaboost) of Cervical Cancer Risk Classification Notebook  I am  getting an accuracy of 99%+. Can you assign this to me?

**Comment by utkarsh0702** [comment]
@nash1o7 Is this training accuracy or validation accuracy?

**Comment by nash1o7** [comment]
@utkarsh0702 for both training & test dataset

**Comment by utkarsh0702** [comment]
@nash1o7 Assigning it to you.
